pkg/cli/webhook/webhook.go

```diff
@@ -102,3 +103,33 @@ func (w *Options) Install(ctx context.Context, providerType string) error {
 	// update repo cr with webhook secret
 	return w.updateRepositoryCR(ctx, response)
 }
+
+func GetProviderName(url string) (string, error) {
+	var (
+		err          error
+		providerName string
+	)
+	switch {
+	case strings.Contains(url, "github"):
+		providerName = "github"
+	case strings.Contains(url, "gitlab"):
+		providerName = "gitlab"
+	case strings.Contains(url, "bitbucket-cloud"):
+		providerName = "bitbucket-cloud"
+	default:
+		msg := "Please enter the provider name to setup the webhook:"
```

Comment:
will this gonna be a dropdown? or enter text

Reply:
Not sure like someone can give URL other than `github`, `gitlab` and `bitbucket cloud`
But added this default case considering that

Reply:
`Please select the git platform target  to setup webhook`

don't we want to use git platform everywhere instead of provider? cc @siamaksade